src/platform/api/controllers/group-booking-controller.js

```
@@ -407,6 +409,99 @@ class GroupBookingController {
+      );
+
+      if (
+        user &&
+        (await PermissionsService._allowUpdate(
+          groupBooking,
+          user.id,
+          tenantId,
+          RolePermission.MANAGE_BOOKINGS,
+        ))
+      ) {
+        const invoiceApp = await TenantManager.getTenantApp(tenantId, "invoice");
```

ESLint

[error] Replace `tenantId,·"invoice"` with `⏎··········tenantId,⏎··········"invoice",⏎········`

(prettier/prettier)